lua/stand.lua

```diff
@@ -44,7 +45,37 @@ function M.setup(options)
 
 		timer:stop()
 		timer:again()
-	end, {})
+	end, { "Stand, dismissing any notifications and restarting the timer." })
+
+	vim.api.nvim_create_user_command("StandEvery", function(opts)
+		options.minute_interval = tonumber(opts.args) * 60000
```

Comment:
Can we break this out to match how we converted to minutes above?

```suggestion
		options.minute_interval = tonumber(opts.args) * 60 * 1000
```